cla_public/templates/_phase-tag.html

```diff
@@ -9,6 +9,19 @@
       Beta
     </strong>
     -->
+    {% if config["CLA_ENV"] == "dev" %}
+      <a class="govuk-link" href="/start">
+        <strong class="govuk-tag govuk-phase-banner__content__tag">
+          Development
+        </strong>
+      </a>
+    {% elif config["CLA_ENV"] == "staging" %}
+      <a class="govuk-link" href="/start">
+        <strong class="govuk-tag govuk-phase-banner__content__tag">
+          Staging
+        </strong>
+      </a>
+    {% endif %}
```

Comment:
How feasible would it be to make this a template fragment we could include here and in the other place we use it? Having it written twice (I think they're identical, right?) isn't the end of the world, but it feels a bit sub-optimal if we expect them not to diverge.

Reply:
I created a new .html to try and act as a fragment, but it seems that the template folder is somehow being ignored? It is not getting picked up in git or my git IDE. Currently looks like I will have to workaround and I may just leave it out

Reply:
Which template folder is that?